src/Stripe.net/Entities/StripeList.cs

```diff
@@ -9,21 +8,34 @@ namespace Stripe
     [JsonObject]
     public class StripeList<T> : StripeEntity, IHasObject, IEnumerable<T>
     {
+        /// <summary>
+        /// A string describing the object type returned.
+        /// </summary>
         [JsonProperty("object")]
         public string Object { get; set; }
 
+        /// <summary>
+        /// A list containing the actual response elements, paginated by any request parameters.
+        /// </summary>
         [JsonProperty("data", ItemConverterType = typeof(StripeObjectConverter))]
         public List<T> Data { get; set; }
 
         /// <summary>
-        /// This field is only valid in the context of paging.
+        /// Whether or not there are more elements available after this set. If <code>false</code>,
+        /// this set comprises the end of the list.
         /// </summary>
         [JsonProperty("has_more")]
         public bool HasMore { get; set; }
 
+        /// <summary>
+        /// This property is deprecated and will not be returned by the API in most cases.
+        /// </summary>
         [JsonProperty("total_count")]
-        public long TotalCount { get; set; }
+        public long? TotalCount { get; set; }
```

Comment:
Can we remove TotalCount entirely instead? We really want to stop supporting this.

Reply:
Sure!